### PR TITLE
Added CloudFlare documentation

### PR DESCRIPTION
**Pull request by ash123456789**
Since we are potentially moving to CloudFlare so that we can manage domain DNS in a centralised location and allow people to have more control over their own projects to reduce reliance on Jared, he has asked me to throw together some documentation for CloudFlare and the basic overview of its features.

There is still a bit to add in regards to Page Rules, Email Address Obfuscation, Hotlink Protection, etc. but for now the basics will do since those options will rarely be accessed.

Would be worth @DivineOmega having a look through this since he is familiar with CloudFlare and could expand upon points within the documentation.

### CODE REVIEW

**Comment by wheatleyjj** [comment]
Fluffing great effort. Some really good content there.

File: platforms/cloudflare/crypto.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Crypto
+
+The crypto section is used for everything relating to SSL certificates, TLS and HTTPS rewriting.
+
+Click on the Crypto tab to get started: 
+
+![Crypto tab](images/crypto/crypto-tab.jpg "Crypto tab")
```
Comment (dextermb) [inline platforms/cloudflare/crypto.md]: How does this work? Isn't this just an image?

**Comment by DivineOmega** [comment]
Just moved the CloudFlare documents link to within the Web Applications page, and did a bit of minor restructuring.

File: platforms/cloudflare/crypto.md (hunk 1 above)
Comment (ash123456789) [reply platforms/cloudflare/crypto.md]: It shows the tab to click on while in the dashboard. It is obvious but there's no reason why it couldn't be there.

**Comment by wheatleyjj** [comment]
Good Addition.

File: platforms/cloudflare/crypto.md (hunk 1 above)
Comment (dextermb) [reply platforms/cloudflare/crypto.md]: Ah, maybe I misread it then. I hadn't seen a mention of a dashboard in any of the documentation 📄

File: platforms/web/cloudflare/caching.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Caching
+
+The Caching tab allows you to purge assets, set the caching level as well as enable or disable development mode.
+
+Click on the Caching tab to get started:
```
Comment (dextermb) [inline platforms/web/cloudflare/caching.md]: I think this should be reworded to:

> Go to the Caching tab to get started

File: platforms/web/cloudflare/crypto.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+# Crypto
+
+The crypto section is used for everything relating to SSL certificates, TLS and HTTPS rewriting.
+
+Click on the Crypto tab to get started:
```
Comment (dextermb) [inline platforms/web/cloudflare/crypto.md]: I think this should be reworded to:

> Go to the Crypto tab to get started

File: platforms/web/cloudflare/dns-management.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# DNS Management
+
+The DNS section provides a fast and really easy interface to update your DNS records. 
+
+Click on the DNS tab to get started:
```
Comment (dextermb) [inline platforms/web/cloudflare/dns-management.md]: I think this should be reworded to:

> Go to the DNS tab to get started

File: platforms/web/cloudflare/speed.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+# Speed
+
+The Speed page on CloudFlare allows you manage asset minification and compression.
+
+Click on the Speed tab to get started:
```
Comment (dextermb) [inline platforms/web/cloudflare/speed.md]: I think this should be reworded to:

> Go to the Speed tab to get started

**Comment by ash123456789** [comment]
@dextermb updated the text - happy with that?

**Comment by DivineOmega** [comment]
Great. I'll get this merged now. :+1:

**Pull request merged**